packages/sdk-core/src/modules/database/BacktraceDatabase.ts

```diff
@@ -246,6 +249,12 @@ export class BacktraceDatabase implements BacktraceModule {
         }
     }
 
+    private deleteRecords(recordsToDelete: BacktraceDatabaseRecord[]) {
+        for (const recordToDelete of recordsToDelete) {
+            this._storageProvider.delete(recordToDelete);
```

Comment:
Why not call `remove` here?

Reply:
I forgot about remove - changing to remove